test1.js

```diff
@@ -3,3 +3,9 @@
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+var foo3 = '' ;
+
+if(foo3  === "bar3") {
```

Comment:
Codacy has a fix for the issue: [Multiple spaces found before '==='.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26726)
```suggestion
if(foo3 === "bar3") {
```